src/main/java/name/webdizz/sonar/grammar/spellcheck/GrammarDictionaryLoader.java

```diff
@@ -49,9 +49,9 @@ public SpellDictionary load() {
                         InputStream inputStream = this.getClass().getResourceAsStream(dictionaryPath);
 
                         BufferedReader dictionaryReader = null;
-                        dictionaryReader = Files.newReader( new File( dictionaryPath ), Charsets.UTF_8 );
+                        dictionaryReader = new BufferedReader(new InputStreamReader(inputStream, "UTF-8"));

```

Comment:
guava has no elegant solution. All options that I found create a lot of code.
For example create file from InputStream and then use Files.newReader(). 
it would be normal for this implementation?